src/pages/workspace/WorkspacePageWithSections.tsx

```diff
@@ -123,6 +107,14 @@ function WorkspacePageWithSections({
         fetchData(shouldSkipVBBACall);
     }, [shouldSkipVBBACall]);
 
+    const shouldShow = useMemo(() => {
+        if (isEmptyObject(policy)) {
+            return true;
+        }
+
+        return !PolicyUtils.isPolicyAdmin(policy) || PolicyUtils.isPendingDeletePolicy(policy);
+    }, [policy]);
```

Comment:
NAB: IMO the code here don't justify a `useMemo` as it is very lean. Just a couple of `!==` checks and `Object.keys(obj ?? {}).length === 0`. I think `useMemo` makes the code less readable and maybe even slower here :P